akka-stream/src/main/scala/akka/stream/impl/fusing/GraphStages.scala

```diff
@@ -37,8 +37,8 @@ object GraphStages {
    * INERNAL API
    */
   private[stream] abstract class SimpleLinearGraphStage[T] extends GraphStage[FlowShape[T, T]] {
-    val in = Inlet[T]("in")
-    val out = Outlet[T]("out")
+    val in = Inlet[T](toString + ".in")
+    val out = Outlet[T](toString + ".out")
```

Comment:
That looks great! :+1:

On Tue, Dec 8, 2015 at 3:04 PM, Martynas Mickevičius <
notifications@github.com> wrote:

> In akka-stream/src/main/scala/akka/stream/impl/fusing/GraphStages.scala
> <https://github.com/akka/akka/pull/19110#discussion_r46955700>:
>
> > @@ -37,8 +37,8 @@ object GraphStages {
> >     * INERNAL API
> >     */
> >    private[stream] abstract class SimpleLinearGraphStage[T] extends GraphStage[FlowShape[T, T]] {
> > -    val in = Inlet[T]("in")
> > -    val out = Outlet[T]("out")
> > +    val in = Inlet[T](toString + ".in")
> > +    val out = Outlet[T](toString + ".out")
>
> Yes, it does (class TakeWithin[T](timeout: FiniteDuration) extends
> SimpleLinearGraphStage[T]).
>
> wiring MaybeSource.out (2145a4f4) -> IdleInject.in (0435d8d4)
> wiring IdleInject.out (5d67b4c9) -> TakeWithin.in (7ed139c8)
> wiring TakeWithin.out (2ec5b7f7) -> headOption.in (562fb04d)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akka/akka/pull/19110/files#r46955700>.
>



-- 
Cheers,
√